mllib/src/main/scala/org/apache/spark/ml/classification/Classifier.scala

```diff
@@ -109,7 +109,7 @@ abstract class Classifier[
       case None =>
         // Get number of classes from dataset itself.
         val maxLabelRow: Array[Row] = dataset.select(max($(labelCol))).take(1)
-        if (maxLabelRow.isEmpty) {
+        if (maxLabelRow.isEmpty || maxLabelRow(0).get(0) == null) {
```

Comment:
Hi, @tovbinm .
If this is a correctness issue, could you add a new test case for your new contribution?

Reply:
@dongjoon-hyun done.